tests/scripts/all.sh

```diff
@@ -3687,18 +3687,29 @@ component_test_psa_crypto_config_accel_cipher_aead () {
     not grep mbedtls_chachapoly library/chachapoly.o
     not grep mbedtls_cmac library/cmac.o
 
+    make
+
```

Comment:
I think this is redundant since we've built everything at L3678.

Reply:
Yes, you are absolutely right. Sorry for that